easybuild/easyblocks/a/abaqus.py

```diff
@@ -64,32 +64,29 @@ def configure_step(self):
             setvar('NOLICENSECHECK', 'true')
         else:
             self.replayfile = os.path.join(self.builddir, "installer.properties")
-            txt = '\n'.join([
-                "INSTALLER_UI=SILENT",
-                "USER_INSTALL_DIR=%s" % self.installdir,
-                "MAKE_DEF_VER=true",
-                "DOC_ROOT=UNDEFINED",
-                "DOC_ROOT_TYPE=false",
-                "DOC_ROOT_ESCAPED=UNDEFINED",
-                "ABAQUSLM_LICENSE_FILE=@abaqusfea",
-                "LICENSE_SERVER_TYPE=FLEXNET",
-                "PRODUCT_NAME=Abaqus %s" % self.version,
-                "TMPDIR=%s" % self.builddir,
-                "INSTALL_MPI=1",
-            ])
+            txt = '\n'.join(
+                [
+                    "INSTALLER_UI=SILENT",
+                    "USER_INSTALL_DIR=%s" % self.installdir,
+                    "MAKE_DEF_VER=true",
+                    "DOC_ROOT=UNDEFINED",
+                    "DOC_ROOT_TYPE=false",
+                    "DOC_ROOT_ESCAPED=UNDEFINED",
+                    "ABAQUSLM_LICENSE_FILE=@abaqusfea",
+                    "LICENSE_SERVER_TYPE=FLEXNET",
+                    "PRODUCT_NAME=Abaqus %s" % self.version,
+                    "TMPDIR=%s" % self.builddir,
+                    "INSTALL_MPI=1",
+                ]
+            )
             write_file(self.replayfile, txt)
 
     def install_step(self):
         """Install ABAQUS using 'setup'."""
         if LooseVersion(self.version) >= LooseVersion('2016'):
             change_dir(os.path.join(self.cfg['start_dir'], '1'))
-            qa = {
-                "Enter selection (default: Install):": '',
-            }
-            no_qa = [
-                '___',
-                '\(\d+ MB\)',
-            ]
+            qa = {"Enter selection (default: Install):": ''}
+            no_qa = ['___', r'\(\d+ MB\)']
```

Comment:
This is going backwards, not forwards imho... ;)

Reply:
That's always the problem with auto-formatters: You'll hit situations where manual formatting would be better. As an alternative to black I'd suggest yapf which allows for more fine grained control over formatting similar to clang-format. Anyhow formatting will always be opinionated so just decide on 1 auto-formatter and configuration and get over it. ;)